metador_core/widget/common.py

```diff
@@ -208,10 +210,16 @@ class Plugin(FileWidget.Plugin):
     @overrides
     def show(self) -> Viewable:
         return self.PANEL_WIDGET[self._meta.encodingFormat](
-            self.file_url(), width=self._w, height=self._h
+            self.file_url(),
+            width=self._w,
+            height=self._h,
+            alt_text="Sorry. Something went wrong while loading the resource",
         )
 
 
+# switching to HTML based audio/video panes because one from Panel not working
```

Comment:
"switching" happens once, you should not write that in a comment that remains in the code. better: "Using ..."